packages/atlas/src/hooks/useVideosInCategories.ts

```diff
@@ -0,0 +1,14 @@
+import { useCategories } from '@/api/hooks/categories'
+import { VideoCategoryData } from '@/config/categories'
+
+export const useVideosInCategories = () => {
```

Comment:
Let's rename this hook, it's not clear what it does. `useVideosCountForCategory`?

And maybe it accepts category ID right away and returns memoized count instead of a function?